docs/source/tutorials.rst

```diff
@@ -0,0 +1,82 @@
+Tutorials
+=========
+
+Here, we provide a small set of tutorials to help ypu get started with lwreg. 
+Please keep in mind that lwreg allow for problem dependent customization:
+The tutorials are meant as a starting point, but might not be applicable to your specific use case.
+
+Setting up a Database and First Registration
+---------------------------------------------
+For a single user, SQLite is the easiest way to get started with lwreg.
+In this first example, we will show how to set up an lwreg instance for registering molecules and querying them.
+We will also see how to retrieve the registerd structures::
```

Comment:
The way the example is presented makes it look like it can be copied and pasted directly into a notebook or Python script.

It can't. The call to initdb requires user intervention.

I would suggest breaking this into two blocks with the first ending at hte call to initdb